backend/de.metas.contracts/src/main/java/de/metas/contracts/FlatrateTermRequest/ModularFlatrateTermRequest.java

```diff
@@ -37,7 +39,9 @@
 public class ModularFlatrateTermRequest
```

Comment:
as it is used, IMHO this class shall be called ModularFlatrateTermQuery...

Reply:
renamed